code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -320,6 +320,9 @@
 				if(91.01 to INFINITY)
 					msg += "[t_He] [t_is] a shitfaced, slobbering wreck.\n"
 
+	if(!skipface && mind && mind.has_antag_datum(ANTAG_DATUM_THRALL))
```

Comment:
replace the mind check with the antag defines, should be ```IS_THRALL``` after you capitalized it